### ISSUE TITLE
The website is down

### ISSUE THREAD

**Issue by springdo** [openpracticelibrary/openpracticelibrary#748]
The OpenPracticeLibrary is currently down when being accessed from the site's main (https://openpracticelibrary.com/) or staging subdomains (https://staging.openpracticelibrary.com/)

I can see the Netlify hosted version of the site is running correct and accessible via the not custom domain entry.
https://openpracticelibrary.netlify.com/

A Whois (http://whois.domaintools.com/openpracticelibrary.com) shows the site may have expired 2 days ago which could explain why in Netlify's Custom domain settings we see some orange issues.

![image](https://user-images.githubusercontent.com/6122439/60010836-826a0300-9670-11e9-8c4a-9fae29eeee14.png)

I've reached out to @sherl0cks  and @rdebeasi  to see who purchased the library domain so we can 
- [ ] Renew it (if needed)
- [ ] Transfer the ownership to the openpracticelibrary mail account (which we can setup a fwd to others for future reference)
- [ ] Consider moving the Domain to AWS or somewhere else were we are managing other domain entries

FYI - @tdbeattie @mtakane @openpracticelibrary/moderators @openpracticelibrary/content-writers

**Comment by rdebeasi** [comment]
I looked into this and the domain has indeed expired. I've renewed it and the site is back online. I can [access the site via webpagetest](http://webpagetest.org/result/190624_XQ_0e88301217b0f5080ded7817fff1f8f8/). Looks like it was set to auto renew, but for whatever reason the auto renewal didn't go through.

Namecheap lets you add admins to an account. I don't think I have permission to do so, but I think @sherl0cks does.

Would it be possible to transfer this domain to Red Hat's ownership? It looks like they use a registrar called [Com Laude](https://www.comlaude.com/). Also, could someone please reimburse me for the $13.16 renewal fee? Feel free to contact me via email or WhatsApp to coordinate that.

Thanks!

**Comment by rdebeasi** [comment]
Also, our forwardmx.io account will probably need to be renewed in the next month or two. @mtakane owns that.

**Comment by sherl0cks** [comment]
Reposting here what I sent Donal in LinkedIn. I did purchase the original account. It was set to auto renew but the card it used is expired. I added @mtakane as an admin way back. If I need to add others in that role, please let me know. At this stage, admin responsibilities for this community is not something I am able to fulfill, so please let me know what change of ownership activities can / need to be done.

**Comment by oybed** [comment]
@sherl0cks thanks for your support on this. If you could add me in as an admin, I'll be happy to process the move to where we keep our other domains to have a one-stop-place for renewals and group management of the domains.

**Comment by sherl0cks** [comment]
Ownership of the domain has been successfully transferred to @oybed

**Comment by rdebeasi** [comment]
Awesome! I checked the forwardmx.io account; it's paid up through July 30th and should auto-renew on @mtakane's card. 

Can we close this issue?

Thanks, everyone!

**Comment by oybed** [comment]
@rdebeasi @sherl0cks do any of you (or someone else from the Labs team) have access to the DNS service used for this domain? I.e.: nsone.net / NS1 - https://ns1.com? If you have access, can you please grant me access to this as well?

**Comment by sherl0cks** [comment]
My understanding was that namecheap was handling this, but to be honest, my understanding of networking is pretty darn basic.

**Comment by rdebeasi** [comment]
These DNS servers are managed through Netlify. I've emailed you a secure link containing the password for @openpracticelibrary-bot. This GitHub account has access to manage the site on http://app.netlify.com.

I had trouble logging into GitHub with this account - I got a "405 Not Allowed" error - but this may be an issue on my end or a temporary problem. I believe @springdo also has these credentials.

I hope that helps!
